vllm/executor/ray_distributed_executor.py

```diff
@@ -340,6 +340,8 @@ def sort_by_driver_then_worker_ip(item: RayWorkerMetaData):
             and v not in self.non_carry_over_env_vars
         ]
 
+        env_vars_to_copy.extend(current_platform.additional_env_vars)
```

Comment:
i'm fine with directly adding it here, without introducing it in the `platform` abstraction, since these env vars do not exist in other platforms.

Reply:
@richardsliu can you fix the comment? I don't want the `platform` abstraction being bloated.